src/Tests/Cask.Tests/CSharpCaskTests.cs

```diff
@@ -68,6 +81,15 @@ public bool IsCask(string key)
                     + string.Join(Environment.NewLine, checks.Select(c => $"  {c.name} -> {c.value}")));
             }
 
+            if (keyBytes == null && result == true)
+            {
+                // See comment above for more information on this condition.
+                Assert.Fail(
+                    "Expected a failure condition as input key bytes did strictly conforming to input key string."
```

Comment:
```suggestion
                    "Expected a failure result as the input string contains invalid base64url characters."
```
Or some other re-wording, I can't decipher this as-is.